public/src/containers/quote.jsx

```diff
@@ -22,11 +31,26 @@ class Quote extends React.Component {
       .catch(err => console.err('Error getting quote: ', err));
   }
 
+  createQuoteInterval() {
+    const intervalId = setInterval(() => {
+      this.setState({
+        quote: null,
+        author: null
+      });
+      this.getQuote();
+    }, 5000);
+    this.setState({ intervalId });
+  }
+
   render() {
     return (
       <div>
-        <div>{this.state.quote}</div>
-        <div>{this.state.author}</div>
+        <div className={this.state.quote ? 'quote-box-visible' : 'quote-box-invisible' }>
+          <div className="quote">
+            <h3>{this.state.quote ? `"${this.state.quote}"` : ''}</h3>
```

Comment:
Since we are displaying movie quotes in this element, we need to wrap them in quotation marks. In order to keep the string concatenation a little cleaner, we used template syntax instead of {'"' + this.state.quote + '"'}. Well, it's cleaner to us. Maybe that's subjective.